code/modules/paperwork/paper.dm

```diff
@@ -569,6 +569,10 @@
 	name = "crumpled note"
 	info = "<b>there is cotten candy in the walls</b>"
 
+/obj/item/paper/bigred/lambda
+	name = "ripped diary entry"
+	info = "Director Smith's Diary \n Entry Date: 15 December 20## \n Today.. I feel progress! The XX-121 species reproduction programm is in full effect and Administrator Cooper gave us the all clear to continue breeding them. It appears that this original XX-121 specimen is what we can call a 'Queen' of the hive... it's almost 5 metres tall and has already produced a bunch of eggs! This beauty will be the next Synthetic, we'll show those Seegson bastards."
```

Comment:
Did you run this by the lore team? If not, reach out to them so they can give some input

Reply:
Well I pinged the Lore Coordinator, I can't ping any lore contributors so I guess we will have to wait and see.